app/static-pages/en/index.mdx

```diff
@@ -18,9 +23,49 @@ export const contentId = "home";
   example2Headline="Use powerful customizations"
   example2Description="With the help of custom filters and data segmentation, even complex issues can be visualized."
 />
-<Contribute
-  headline="Would you like to visualize your own data?"
-  description="Find out how you can integrate your data into the LINDAS Linked Data Service."
-  buttonLabel="Learn how"
-  buttonUrl="https://lindas.admin.ch/?lang=en"
-/>
+<Section>
+  <Contribute
+    headline="Would you like to visualize your own data?"
+    description="Find out how you can integrate your data into the LINDAS Linked Data Service."
+    buttonLabel="Learn how"
+    buttonUrl="https://lindas.admin.ch/?lang=en"
+  />
+  <div style={{ width: "1px", backgroundColor: "#e5e5e5" }}></div>
+  <Newsletter
+    headline="Subscribe to our Newsletter"
+    description="Stay up to day and subscribe to our newsletter by adding your email address below."
+    buttonLabel="Subscribe"
+    buttonUrl="mailto:visualize@bafu.admin.ch"
+  />
+</Section>
+<Section
+  sx={{
+    color: "gray.800",
```

Comment:
I am not sure if this works, in the theme we use "grey" key. Maybe there's a default `gray` palette that comes from MUI, but I am pretty sure it should be `grey.800` (also in other places) 🤔